include/z64animation.h

```diff
@@ -9,16 +9,17 @@
 struct PlayState;
 struct Actor;
 struct SkelAnime;
+struct PlayerAnimationFrame;
 
 #define LINK_ANIMETION_OFFSET(addr, offset) \
-    (SEGMENT_ROM_START(link_animetion) + ((u32)addr & 0xFFFFFF) + ((u32)offset))
+    (SEGMENT_ROM_START(link_animetion) + ((uintptr_t)addr & 0xFFFFFF) + ((uintptr_t)offset))
 #define LIMB_DONE 0xFF
 #define ANIMATION_ENTRY_MAX 50
 
 #define ANIM_FLAG_UPDATEY   (1 << 1)
 #define ANIM_FLAG_NOMOVE    (1 << 4)
 
-typedef enum {
+typedef enum AnimationModes {
```

Comment:
```suggestion
typedef enum AnimationMode {
```
Probably should be singular like the limb enums, same for the struct name as well below.